tests/test_parking_lot.py

```diff
@@ -1,16 +1,206 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+import datetime
+import random
+import unittest
+from parking_lot.parking_lot import (
+    Parking_lot, Full_parking_error, Car, Cannot_find_car_error
+)
+from parking_lot.factories import (
```

Comment:
is module factories only used for test purposes? if yes, why not to relocate the file to tests folder to don't distribute code intended for test code to client side.

Reply:
i normally leave the factories in the package because normally i need those in another projects, but i can do the change.